.github/workflows/api-docs.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: |
           git checkout -b ${DOC_BRANCH}
           python3 scripts/gen_vimdoc.py
-          printf '::set-output name=UPDATED_DOCS::%s\n' $([ -z "$(git diff)" ]; echo $?)
+          printf 'UPDATED_DOCS=%s\n' $([ -z "$(git diff)" ]; echo $?) >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          printf 'UPDATED_DOCS=%s' $([ -z "$(git diff)" ]; echo $?) >> $GITHUB_OUTPUT
```

Reply:
although, why is this done backwards?
```sh
	if ! git diff --quiet then
		echo 'UPDATED_DOCS=1' >> $GITHUB_OUTPUT
	fi
```

Reply:
I'm actually not sure if `\n` is required for `printf` here, but either way, I generally think it's better to not rely on `$?` if it can be helped.

Reply:
Nah, I don't wanna mess with these because then I'd be forced to take responsibility for them lol.